tests/test_PostageStampMaker.py

```diff
@@ -90,5 +95,27 @@ def test_create_sequence_function(self):
         for stamp in my_stamps:
             self.assertTrue(isinstance(stamp, afwImage.ExposureF))
 
+    def test_hdu_from_exposure(self):
+        """
+        Test the conversion of the image from Exposure object directly
+        to an HDU.
+        """
+        my_stamp = self.stamp_maker.create(self.ra, self.dec, self.size)
+        my_hdu = convert_image_to_hdu(my_stamp)
+
+        # Write the Exposure to a FITS file to test the image data and
+        # WCS keywords in the in-memory HDU against the versions
+        # written by the Stack.
+        stamp_file = 'temp_stamp.fits'
+        my_stamp.writeFits(stamp_file)
+        stack_hdu = fits.open(stamp_file)[1]
+
+        for value1, value2 in zip(my_hdu.data.flatten(),
```

Comment:
Note that numpy has an `array_equal` function, so you could say: `self.assertTrue(numpy.array_equal(my_hdu.data, stack_hdu.data)`.

Another option is to say `self.assertTrue((my_hdu.data==stack_hdu.data).all())`